libraries/lib-music-information-retrieval/MirUtils.cpp

```diff
@@ -0,0 +1,80 @@
+#include "MirUtils.h"
+
+#include <cmath>
+#include <numeric>
+
+namespace MIR
+{
+namespace
+{
+constexpr auto bpmStdDev = 29.7953;
```

Comment:
Should this standard deviation be defined in the same place as the mean, `bpmExpectedValue`?

Reply:
These will be subject to change in follow-up improvement PRs, I'm reluctant to invest time centralizing them, as little work as that may be.